ckanext/syndicate/controllers/syndicate.py

```diff
@@ -0,0 +1,107 @@
+import ckan.lib.base as base
+import ckan.lib.helpers as h
+from ckan.common import response, c, _
+import ckan.model as model
+import ckan.plugins.toolkit as tk
+from ckan.logic import NotAuthorized, NotFound
+from sqlalchemy.orm.exc import NoResultFound
+from ckanext.syndicate.plugin import syndicate_dataset
+
+import json
+
+abort = base.abort
+
+
+def _get_tasks(group_id):
+    tasks = []
+
+    try:
+        tasks = model.Session.query(model.TaskStatus).join(
+            model.Package, model.TaskStatus.entity_id == model.Package.id
+        ).join(
+            model.Group, model.Package.owner_org == model.Group.id
+        ).filter(model.Group.id == group_id).all()
+    except NoResultFound:
```

Comment:
You probably had some other code here, because Query.all won't raise `NoResultFound`